File: Service_eHospital_DongNai_A_Config/Implement/Sys_ModifySource_ServerService.cs
namespace Service_eHospital_DongNai_A_Config.Implement
{
    public class Sys_ModifySource_ServerService : BaseService<Sys_ModifySource_Server, ISys_ModifySource_ServerRepository>
    , ISys_ModifySource_ServerService
    {
    private readonly ISys_ModifySource_ServerRepository _Sys_ModifySource_ServerRepository;
    public Sys_ModifySource_ServerService(ISys_ModifySource_ServerRepository Sys_ModifySource_ServerRepository) : base(Sys_ModifySource_ServerRepository)
    {
    _Sys_ModifySource_ServerRepository = Sys_ModifySource_ServerRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A_Config/Sys_ModifySource_Server.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_ModifySource_Server } from './Sys_ModifySource_Server.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_ModifySource_ServerService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'ID', 'DateUpdate', 'VersionNo', 'FileName', 'PathName', 'Size', 'ModifyDate'];

    List: Sys_ModifySource_Server[] | undefined;
    ListFilter: Sys_ModifySource_Server[] | undefined;
    FormData!: Sys_ModifySource_Server;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_ModifySource_Server";
    }
}

